cabal v1-install fails after tidal-1.7.1 update

With `cabal v1-install` I get this error:
```
<no location info>: warning: [-Wmissing-home-modules]
    These modules are needed for compilation but not listed in your .cabal file's other-modules: 
        Code Parameters
[6 of 7] Compiling Edit             ( src/Edit.hs, dist/build/feedforward/feedforward-tmp/Edit.o )

src/Edit.hs:526:41: error:
    Ambiguous occurrence ‘resolve’
    It could refer to
       either ‘Sound.Tidal.Context.resolve’,
              imported from ‘Sound.Tidal.Context’ at src/Edit.hs:30:1-54
              (and originally defined in ‘Sound.Tidal.Stream’)
           or ‘Edit.resolve’, defined at src/Edit.hs:533:1
    |
526 |                          remote_addr <- resolve "127.0.0.1" "57110"
    |                                         ^^^^^^^

src/Edit.hs:987:27: error:
    Ambiguous occurrence ‘resolve’
    It could refer to
       either ‘Sound.Tidal.Context.resolve’,
              imported from ‘Sound.Tidal.Context’ at src/Edit.hs:30:1-54
              (and originally defined in ‘Sound.Tidal.Stream’)
           or ‘Edit.resolve’, defined at src/Edit.hs:533:1
    |
987 |            remote_addr <- resolve "127.0.0.1" "57110"
    |                           ^^^^^^^
cabal: Leaving directory '.'
cabal: Error: some packages failed to install:
feedforward-0.0.1-1DP1FutITQf9j3PFVRqcEK failed during the building phase. The
exception was:
ExitFailure 1
```

--- a/src/Edit.hs
+++ b/src/Edit.hs
@@ -27,7 +27,7 @@ import           Data.Time.Format
 import qualified Network.Socket          as N
 import qualified Network.WebSockets      as WS
 import           Sound.OSC.FD
-import           Sound.Tidal.Context     hiding (when)
+import           Sound.Tidal.Context     hiding (when, resolve)
 import           Sound.Tidal.Tempo       (timeToCycles)
 import           System.Directory
 import           System.Environment      (getArgs, lookupEnv)

